docs/UserGuide.md

```diff
@@ -378,22 +377,22 @@ Format: `incompleteorders`
 
 Sorts all orders based on a chosen field and arrangement.
 
-Format: `sortorders f/FIELD o/ORDERING`
+Format: `sortorders f/FIELD [o/ORDERING]`
 
 * Your orders are sorted based on the `FIELD` chosen. You may choose between:
   * The Date field, identified with a "d" or "date".
   * The Amount field, identified with an "a" or "amount".
-* By default, the orders are sorted in the sequence they are added. This is also used as a tiebreaker when two or more orders have an identical value for the `FIELD` chosen.
 * The direction of the arrangement depends on the `ORDERING`, which is either:
   * Ascending, identified with an "asc" or "ascending".
   * Descending, identified with a "desc" or "descending".
+* If the `ORDERING` parameter is not supplied, the list will be sorted in ascending order.
 
 <div markdown="block" class="alert alert-primary">
 
-**:information_source: Note:**<br>
+**:information_source: Note:** <br>
 
+* By default, your orders are sorted in the sequence they are added. This is also used as a tiebreaker when two or more orders have an identical value for the `FIELD` chosen.
```

Comment:
Sorry i'm not sure if i'm misunderstanding, but 388 says the default is ascending order, and 394 says the default is sequence they are added, and I thought it might be a little unclear. I assume what you meant is that when you call sortorders, then default sort is ascending according to **amount**, and normally when the list is displayed, they are displayed in the order they were added. Editing the wording might make this a little clearer.

Reply:
Thanks for the suggestion! I've tried to make it clearer